openadapt/strategies/mixins/freight_documents_test/Screenshot Dataset.v1i.coco/README.dataset.txt

```diff
@@ -0,0 +1,6 @@
+# Screenshot Dataset > 2023-06-03 1:20pm
```

Comment:
Can you please move this (and other similar files) into the `resources` directory?